blocks/api/serializer.js

```diff
@@ -111,6 +111,11 @@ export function serializeAttributes( attrs ) {
 
 export function serializeBlock( block ) {
 	const blockName = block.name;
+
+	if ( 'core/nextpage' === blockName ) {
+		return '<!--nextpage-->';
+	}
+
```

Comment:
is that problematic then that the block doesn't do it. maybe we've surfaced an oddity or bug?

it seems like other blocks might also have no attributes and if they break here we will want to make sure that doesn't happen.